libs/spaces/src/lib/spaces-status.service.ts

```diff
@@ -43,16 +48,54 @@ export class SpacesStatusService extends BaseClass {
         this._init();
     }
 
+    /** Manual removal of a booking from store */
+    public removeBooking(system_id: string, booking_id: string){
+        if(!system_id || !booking_id) return;
+        const map = {...this._list_status.getValue()};
+        const bookings:any[] = map[system_id]?.bookings || [];
+        const idx = bookings.findIndex(e => e.id === booking_id);
+        if(idx > -1){ 
+            bookings.splice(idx,1);
+            map[system_id].bookings = bookings;
+            this._list_status.next(map);
+        }
+    }
+
     private async _init() {
-        await this._spaces.initialised.pipe(first()).toPromise();
-        // bind property
+
         this.subscription(
             'space_list',
             this.building_spaces.subscribe((l) => {
-                const spaces = l.map((_) => _.id);
                 this.unsubWith('listen:');
                 this.unsubWith('bind:');
-                spaces.forEach((id) => this.bindTo(id, 'status'));
+                l.forEach((space) => {
+                    const id = space.id
+                    // Status binding
+                    this.bindTo(id, 'status');
+
+                    //Bookings binding
+                    this.bindTo(id, 'bookings', (v:any[]) => {
```

Comment:
Binding to this for all rooms will result in the frontend parsing lots of bookings and can also leak other booking data to the user.